tree/ntuple/v7/src/RField.cxx

```diff
@@ -1649,6 +1652,131 @@ void ROOT::Experimental::RClassField::AcceptVisitor(Detail::RFieldVisitor &visit
 
 //------------------------------------------------------------------------------
 
+TClass *ROOT::Experimental::RField<TObject>::GetClass()
+{
+   return TClass::GetClass("TObject");
```

Comment:
I don't remember right now if consecutive calls to TClass::GetClass will use a cached value. Otherwise since this is only calling it for `TObject` we might want to cache it here.